src/backend/commands/explain.c

```diff
@@ -2668,6 +2668,8 @@ show_sort_info(SortState *sortstate, ExplainState *es)
 	if (!es->analyze)
 		return;
 
+	Assert(((PlanState *) sortstate)->instrument);
```

Comment:
Why did you decide that `((PlanState *) sortstate)->instrument` cannot be NULL? Maybe it would be better to return from a function, as is done a few lines below?

Reply:
It can't be null because it's dereferenced in the next line